doc/changes/changes_3.4.0.md

```diff
@@ -0,0 +1,23 @@
+# Test Database Builder for Java 3.4.0, released 2022-09-30
+
+Code name: Support system properties to set DEBUG_ADDRESS and LOG_LEVEL.
+
+## Summary
+
+This release adds support to use properties to set `DEBUG_ADDRESS` and `LOG_LEVEL` for virtual schemas, please see the [User Guide](../user_guide/user_guide.md#debug-output) for details.
```

Comment:
Relative urls will break on the GH release page. Either change it here to an absolute url or update it later after the release :)